docs/content/latest/explore/multi-region-deployments/read-replicas.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Read replicas in YCQL
+headerTitle: Read replicas in YCQL
+linkTitle: Read replicas in YCQL
+description: Read replicas in YCQL
+menu:
+  latest:
+    name: Read replicas in YCQL
+    identifier: explore-multi-region-deployments-read-replicas
+    parent: explore-multi-region-deployments
+    weight: 720
+isTocNested: true
+showAsideToc: true
+---
+
+
+
+YugabyteDB supports the following types of reads: 
+
+- [Follower reads](../../follower-reads/macos/) that enable spreading the read workload across all data-hosting replicas.
+- Observer reads that use read replicas. The latter obtain their data via [asynchronous replication](../asynchronous-replication-ycql) which allows for the read workload to be offloaded from the primary cluster. Read replicas are created as a separate cluster that may be located in a different region, possibly closer to the consumers of the data which would result in lower-latency access and enhanced support of analytics workloads. 
+
+A universe can have one primary cluster that uses follower reads and several read replica clusters that use observer reads.
+
+Stale reads are possible with an upper bound on the amount of staleness. Reads are guaranteed to be timeline-consistent. You need to set the consistency level to `ONE` in your application to get reads. In addition, you have to set the application’s local universe to the read replica cluster’s region.
+
+## Prerequisites
+
+Ensure that you have downloaded and configured YugabyteDB, as described in [Quick Start](https://docs.yugabyte.com/latest/quick-start/install/macos/).
+
+{{< note title="Note" >}}
+
+This document uses a client application based on the [yb-sample-apps](https://github.com/yugabyte/yb-sample-apps) workload generator.
+
+{{< /note >}}
+
+Since you cannot use read replicas without a primary cluster, ensure that you have the latter available.
```

Comment:
This sentence could be changed to say something like - lets setup a primary cluster and work with follower reads. or something like that.

Reply:
Follower reads are not the subject of this doc (we have a different doc that covers this topic). The only reason we are showing examples of the primary cluster setup is because it's a prerequisite for demonstrating read replicas.
I'm rephrasing a bit:
Also, since you cannot use read replicas without a primary cluster, ensure that you have the latter available. The following command sets up a primary cluster of three nodes in cloud `c`, region `r` and zones `z1`, `z2`, and `z3`: